src/pubsub/ua_pubsub.h

```diff
@@ -293,6 +304,28 @@ UA_ReaderGroup_addSubscribeCallback(UA_Server *server, UA_ReaderGroup *readerGro
 void
 UA_ReaderGroup_subscribeCallback(UA_Server *server, UA_ReaderGroup *readerGroup);
 
+/*********************************************************/
+/*               Reading Message handling                */
+/*********************************************************/
+
+#ifdef UA_ENABLE_PUBSUB_ENCRYPTION
+UA_StatusCode
+verifyAndDecryptNetworkMessage(const UA_Logger *logger,
+                               UA_ByteString *buffer, size_t *currentPosition,
+                               UA_NetworkMessage *currentNetworkMessage,
+                               UA_ReaderGroup *readerGroup);
+#endif
+
+UA_StatusCode
+readNetworkMessage(const UA_Logger *logger,
+                   UA_ByteString *buffer, size_t *currentPosition,
+                   UA_NetworkMessage *currentNetworkMessage,
+                   UA_ReaderGroup *readerGroup);
+
+UA_StatusCode
```

Comment:
Does this only receive the message or also process it?
Add a short comment to help getting a quick overview on the code.